### PR TITLE
Different icon size on connect-with-us page #4306

### PR DESCRIPTION
**Pull request by Manish254252**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Manish254252** [comment]
I have fixed this issue ====> Different icon size on connect-with-us page #4306 .

**Comment by l5io** [comment]
🚀 Preview for commit 08671ce225ef4eab54c3a4b0040c8af496576049 at: https://647f320692530128894cd356--layer5.netlify.app

**Comment by leecalcote** [comment]
Thanks for this, @Manish254252 👍

**Review by thisiskaransgit — APPROVED** [review]
Thank you for working on this, the changes look good 🎯

**Comment by vishalvivekm** [comment]
@Manish254252 Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Comment by Manish254252** [comment]
> @Manish254252 Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

Not sure what you mean ?

**Comment by vishalvivekm** [comment]
> > @Manish254252 Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)
> 
> Not sure what you mean ?

I've listed this pr in websites call's meeting minutes which is held every monday at 5:30 PM IST.
Would you be on call today to discuss this item?

**Comment by thisiskaransgit** [comment]
@Manish254252 merge conflicts

File: src/sections/Home/Partners-home/partnerSection.style.js
Comment (HetviSoni) [inline src/sections/Home/Partners-home/partnerSection.style.js:1]: There are many lines where no change in code is done only white space(1 tab) has been removed.

**Pull request closed without merging**